frappe/database/mariadb/database.py

```diff
@@ -142,12 +142,19 @@ def get_connection_settings(self) -> dict:
 		if frappe.conf.local_infile:
 			conn_settings["local_infile"] = frappe.conf.local_infile
 
-		if frappe.conf.db_ssl_ca and frappe.conf.db_ssl_cert and frappe.conf.db_ssl_key:
-			conn_settings["ssl"] = {
+		# Configure SSL settings
+		if frappe.conf.db_ssl_ca:
+			ssl_config = {
 				"ca": frappe.conf.db_ssl_ca,
-				"cert": frappe.conf.db_ssl_cert,
-				"key": frappe.conf.db_ssl_key,
+				"check_hostname": frappe.conf.get("db_ssl_check_hostname", True),
```

Comment:
Default is `None`, any reason to set it true here?

https://github.com/PyMySQL/PyMySQL/blob/main/pymysql/connections.py#L210

Reply:
Yep, better to stick with defaults from lib. 

Thanks, Updated.